e2e/motoko/Main.mo

```diff
@@ -146,80 +147,128 @@ shared ({ caller = installer }) actor class Main() {
+                                    "0x000000000000000000000000b6bc16189ec3d33041c893b44511c594b1736b8a",
+                                ],
+                            ];
+                        },
+                    ),
+                );
+                Cycles.add(candidRpcCycles);
+                assertOk(
+                    networkName,
+                    "eth_getBlockByNumber",
+                    await canister.eth_getBlockByNumber(services, null, #Latest),
+                );
+                Cycles.add(candidRpcCycles);
+                assertOk(
+                    networkName,
+                    "eth_getTransactionReceipt",
+                    await canister.eth_getTransactionReceipt(services, null, "0xdd5d4b18923d7aae953c7996d791118102e889bea37b48a651157a4890e4746f"),
+                );
+                Cycles.add(candidRpcCycles);
+                assertOk(
+                    networkName,
+                    "eth_getTransactionCount",
+                    await canister.eth_getTransactionCount(
+                        services,
+                        null,
+                        {
+                            address = "0x1789F79e95324A47c5Fd6693071188e82E9a3558";
+                            block = #Latest;
+                        },
+                    ),
+                );
+                Cycles.add(candidRpcCycles);
+                assertOk(
+                    networkName,
+                    "eth_feeHistory",
+                    await canister.eth_feeHistory(
+                        services,
+                        null,
+                        {
+                            blockCount = 3;
+                            newestBlock = #Latest;
+                            rewardPercentiles = null;
+                        },
+                    ),
+                );
+                switch services {
+                    case (#Custom _) {
+                        // Skip sending transaction for custom chains due to chain ID mismatch
+                    };
+                    case _ {
+                        Cycles.add(candidRpcCycles);
+                        assertOk(
+                            networkName,
+                            "eth_sendRawTransaction",
+                            await canister.eth_sendRawTransaction(
+                                services,
+                                null,
+                                "0xf86c098504a817c800825208943535353535353535353535353535353535353535880de0b6b3a76400008025a028ef61340bd939bc2195fe537567866003e1a15d3c71ff63e1590620aa636276a067cbe9d8997f761aecb703304b3800ccf555c9f3dc64214b297fb1966a3b6d83",
+                            ),
+                        );
+                    };
+                };
+            };
+
+            for ((name, services) in allServices.vals()) {
+                pending.add(testCandidRpc(name, services));
+            };
+        };
+
+        for (a in pending.vals()) {
```

Comment:
Single-letter variable alert!   :slightly_smiling_face:

Reply:
Haha, fixed. I was very intentional about removing these from the original Ethereum integration codebase, but in this case I had no idea what to name this and indeed was the culprit behind this one. Going with `awaitable` for now. Thanks for the review!